### DISCUSSION TITLE
How to solve an Nginx rewrite error?

### DISCUSSION THREAD

**Discussion by dirkclemens** [datenstrom/community#888] [Ask a question] [answered]
Dear all, 
I am running the latest Yellow cms with Ubuntu Server 22.04, Nginx 1.18.0, php 8.1 and php8.1-fpm and all necessary php extensions installed.

Even with exact code snippet taken from "Problems with Nginx" (https://datenstrom.se/yellow/help/troubleshooting#problems-with-nginx) and reading [http://nginx.org/en/docs/http/ngx_http_rewrite_module.html](http://nginx.org/en/docs/http/ngx_http_rewrite_module.html) (and Stackoverflow among others) I still get the error message "Datenstrom Yellow requires rewrite support! [See troubleshooting](https://datenstrom.se/yellow/help/troubleshooting)."

My "nginx.conf" (I am using virtual hosts with nginx) contains the code with a small change, because `fastcgi_pass 127.0.0.1:9000;` thows an error, so I changed it to `fastcgi_pass unix:/run/php/php8.1-fpm.sock;` (which exists in this folder).
I also added the following line to my nginx.conf 
``` 
        access_log /var/log/nginx/yellow-access.log;
        error_log  /var/log/nginx/yellow-error.log error;
        rewrite_log on;
        uninitialized_variable_warn on;
```
but no error is logged there.

http://localhost/info.php works fine for sure

Any help appreciated, thanks in advance.

**Comment by annaesvensson** [comment, marked as answer]
Activate the debug mode and see if the URL shown on screen matches with the URL shown in your web browser. The installer checks if the web server is responding, the Nginx `server_name` may not be configured correctly or may not be detected automatically. See also datenstrom/community#225.

Tip: You can activate the debug mode with `CoreDebugMode: 1` in file `system/extensions/yellow-system.ini`.
